tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -257,4 +258,59 @@ var _ = Describe("docker", Ordered, ContinueOnFailure, func() {
 			Expect(libs).To(ContainElements([]string{"libcuda.so", "libcuda.so.1"}))
 		})
 	})
+
+	When("A container tries to create firmware files on the host", Ordered, func() {
+		var tmpDir string
+		BeforeAll(func(ctx context.Context) {
+			_, _, err := runner.Run("docker pull ubuntu")
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		BeforeEach(
+			func(ctx context.Context) {
+				var err error
+				tmpDir, _, err = runner.Run("mktemp -d")
+				Expect(err).ToNot(HaveOccurred())
+				tmpDir = strings.TrimSpace(tmpDir)
```

Comment:
Is there no per-test tmp dir that we can use?

Reply:
This is what the `BeforeEach` is doing: it creates a new tmp dir per test (per `If` block)